psalm.xml

```diff
@@ -37,5 +37,40 @@
 				<referencedClass name="Doctrine\DBAL\Statement" />
 			</errorLevel>
 		</UndefinedDocblockClass>
+		<DeprecatedClass>
+			<errorLevel type="suppress">
```

Comment:
Not sure i get this right, but wouldn't this suppress all of them directly? If we have any we should rather add to the baseline, right?

Reply:
Indeed, I should have checked more thoroughly. Thanks for catching this.